ui/components/MeshConfigComponent.js

```diff
@@ -362,6 +362,7 @@ class MeshConfigComponent extends React.Component {
     }, (result) => {
       this.props.updateProgress({ showProgress: false });
       if (typeof result !== 'undefined') {
+        // Prompt
```

Comment:
@vineethvanga18 Prompt component to be used here

Reply:
@vineethvanga18 @kumarabd does this PR need updates from Vineeth before merging?

Reply:
yes